english/elves-and-shoemaker/elves-and-shoemaker.md

```diff
@@ -0,0 +1,7 @@
+There was once a shoemaker, who worked very hard and was honest, but still he could not earn enough to live upon. At last all he had in the world was gone, save just leather enough to make one pair of shoes.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.